MemoryManagement/src/memorymanagementAlgorithm/Void.java

```diff
@@ -21,55 +28,83 @@ we have intialy void (free locations)= base = 0 limit 1000
 
 but if it's from 500 to 600 and from 900 to 1000
 we will add the segments easy as 2 free segments
+
+
+ */
+/**
+ *
+ * @author ahmed
  */
 public class Void {
+//   Collections.sort(xx,(a, b) -> a.getBase() < b.getBase() ? -1 : a.getBase() == b.getBase() ? 0 : 1);

```

Comment:
this comment is irrelevant here to this section. it must be removed or moved.

Reply:
fixed